<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE TS>
<TS version="2.1" language="az">
    <extra-po-header-language>az</extra-po-header-language>
    <extra-po-header-language_team>Azerbaijani</extra-po-header-language_team>
    <extra-po-header-last_translator>Xəyyam &lt;xxmn77@gmail.com&gt;</extra-po-header-last_translator>
    <extra-po-header-plural_forms>nplurals=2; plural=n != 1;</extra-po-header-plural_forms>
    <extra-po-header-po_revision_date>2020-04-09 19:42+0400</extra-po-header-po_revision_date>
    <extra-po-header-project_id_version></extra-po-header-project_id_version>
    <extra-po-header-x_generator>Lokalize 19.12.3</extra-po-header-x_generator>
    <extra-po-header_comment># Xəyyam &lt;xxmn77@gmail.com&gt;, 2020.</extra-po-header_comment>
    <extra-po-headers>Project-Id-Version,PO-Revision-Date,Last-Translator,Language-Team,Language,MIME-Version,Content-Type,Content-Transfer-Encoding,X-Qt-Contexts,Plural-Forms,X-Generator</extra-po-headers>
<context>
    <name>KAuth::DBusHelperProxy</name>
    <message>
        <location filename="backends/dbus/DBusHelperProxy.cpp" line="74"/>
        <source>DBus Backend error: service start %1 failed: %2</source>
        <translation>DBus daxili xətası: %1 xidmətini başlatmaq mümkün olmadı: %2</translation>
    </message>
    <message>
        <location filename="backends/dbus/DBusHelperProxy.cpp" line="89"/>
        <source>DBus Backend error: connection to helper failed. %1
(application: %2 helper: %3)</source>
        <translation>DBus daxili xətası: köməkçi ilə əlaqə uğursuz oldu. %1
(tətbiq: %2 köməkçi: %3</translation>
    </message>
    <message>
        <location filename="backends/dbus/DBusHelperProxy.cpp" line="124"/>
        <source>DBus Backend error: could not contact the helper. Connection error: %1. Message error: %2</source>
        <translation>Dbus daxili xətası: köməkçiyə qoşulmadı. Əlaqə xətası: %1.Xəta bildirişi: %2</translation>
    </message>
</context>
<context>
    <name>KAuth::ExecuteJob</name>
    <message>
        <location filename="executejob.cpp" line="86"/>
        <source>Tried to start an invalid action</source>
        <translation>Etibarsız tapşırığı başlatmağa cəhd edildi</translation>
    </message>
    <message>
        <location filename="executejob.cpp" line="104"/>
        <source>Unknown execution mode chosen</source>
        <translation>Naməlum icra etmə rejimi seçildi</translation>
    </message>
    <message>
        <location filename="executejob.cpp" line="149"/>
        <source>Unknown status for the authentication procedure</source>
        <translation>Autentifikasiya prosedurunun naməlum vəziyyəti</translation>
    </message>
    <message>
        <location filename="executejob.cpp" line="161"/>
        <source>The current backend only allows helper authorization, but this action does not have a helper.</source>
        <translation>Cari icraata icazə vermək yalnız köməkçinin köməyi ilə mümkündür, ancaq bu həmin fəaliyyət üçün mövcud deyil</translation>
    </message>
    <message>
        <location filename="executejob.cpp" line="169"/>
        <location filename="executejob.cpp" line="193"/>
        <source>The backend does not specify how to authorize</source>
        <translation>Bu icraat necə icazə verilməsini göstərmir</translation>
    </message>
</context>
</TS>
